CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 ==========
 Changelog
 ==========
+1.11.4 (2021-03-22)
```

Comment:
I think it is `1.11.3` isn't it?

Reply:
1.11.4 is on production, I think either 1.11.3 was broken or we forgot to document the changes